oteapi/strategies/download/file.py

```diff
@@ -30,6 +31,10 @@ class FileConfig(BaseModel):
             "Encoding used when opening the file. The default is platform dependent."
         ),
     )
+    datacache_config: Optional[AttrDict] = Field(
+        AttrDict(),
```

Comment:
```suggestion
    datacache_config: Optional[DataCacheConfig] = Field(
        None,
```
This should use the `DataCacheConfig` model instead of `AttrDict`.
It might also be that instead of having `Optional[DataCacheConfig]` this should simply be `DataCacheConfig` with the default value of `DataCacheConfig()` (instead of the suggested `None` here).